### PR TITLE
feat: added a more filters button to the filter form

### PR DESCRIPTION
**Pull request by jadeallencook**
Screenshot 2023-11-28 at 2 24 36 AM

### CODE REVIEW

File: src/utils/filter-requests.ts [hunk 1]
```diff
@@ -4,19 +4,24 @@ import RequestAidInterface from '@interfaces/request-aid';
...
+    return Object.entries(requests).reduce((object, [id, request]) => {
         const { stage, location, language, driver, coordinator } = filters;
+        const string = filters.string?.toLowerCase().trim();
         return stage !== request.stage ||
             (location && location !== request.location) ||
             (language && language !== request.language) ||
             (driver && driver !== request.driver) ||
+            (string &&
+                request.name.toLowerCase().trim().indexOf(string) &&
+                request.email.toLowerCase().trim().indexOf(string)) ||
```
Comment (A1Liu) [inline src/utils/filter-requests.ts:17]: `indexOf` returns `-1` when it doesn't find anything; isn't `-1` truthy?

File: interfaces/filters.ts [hunk 2]
```diff
@@ -9,4 +9,5 @@ export interface FiltersInterface {
     driver: DriverKeyType | '';
     stage: StageKeyType | '';
     coordinator: string;
+    string: string;
```
Comment (A1Liu) [inline interfaces/filters.ts:12]: Nit: might be better to call the field something like `nameSearchQuery`

**Review by A1Liu — APPROVED** [review]

File: interfaces/filters.ts (hunk 2 above)
Comment (jadeallencook) [reply interfaces/filters.ts]: Good call, I changed it to `searchQuery` 👍

File: src/utils/filter-requests.ts (hunk 1 above)
Comment (jadeallencook) [reply src/utils/filter-requests.ts]: That's correct, I updated this to:
`request.name.toLowerCase().trim().indexOf(searchQuery) === -1`

**Pull request merged**